app/avatar/utils.py

```diff
@@ -572,11 +572,6 @@ def svg_to_png(svg_content, width=100, height=100, scale=1, index=None, prefer=N
     png = None
     if not prefer or prefer == 'pyvips':
         png = svg_to_png_pyvips(svg_content, scale=scale)
-    if not png:
-        if not index:
-            index = random.randint(1000000, 10000000)
-        print("failed; using inkscape")
-        return svg_to_png_inkscape(svg_content, height=height, width=width, index=index, extra_flags=extra_flags)
```

Comment:
@thelostone-mc 
This assumes we should always go with the `svg_to_png_pyvips` method of generating PNGs.
In this case we should remove the `if` in line 573